wordpress-mobile/WordPress-Android
# Rearrange packages - new site creation

Fixes #9058

**Note: The branch is based on #9057 -> change target to feature/master-site-creation when #9057 is merged.**

I've moved around files related to new site creation flow. I've done everything using AS tools, so it should be safe. Let me know if you like this structure or you'd prefer to change something. Also let me know, if you don't like moving anything at all, I don't feel strongly about it.

To test:
I think successful build is enough. However, if you want to make sure just try to walk through the site creation flow.


Update release notes:

[ x ] If there are user facing changes, I have added an item to RELEASE-NOTES.txt.
